tv_apps/tv-ftu/js/tv_ftu.js

```diff
@@ -47,26 +105,36 @@ window.addEventListener('load', function() {
     },
 
     onKeyMove: function (key) {
-      switch (key) {
-        case 'up':
-          if (this._currentFocused !== this._termsSection) {
-            this.buttonNav.blur();
-            this.buttonNav.pause();
-            this._termsSection.focus();
-            this.fire('focus', this._termsSection);
-            this._arrowIcon.classList.remove('hidden');
+
+      switch (this._currentNavigationTarget) {
```

Comment:
You have `currentNavigationTarget` and `currentFocused` and didn't have any description between `NavigationTarget` and `Focused`. I'd thought `NavigationTarget` is a kind of HTMLElement or element id at first glance but it seems `NavigationTarget` is just several types of word. I feel it pretty confused. Please try to make your variables more specific. In this case I feel `currentFocusType` or something may be more suitable.